Argo/Extensions/Array.swift

```diff
@@ -0,0 +1,9 @@
+extension Array {
+  func mapThrows<T>(@noescape transform: Generator.Element throws -> T) throws -> [T] {
```

Comment:
I named this differently because naming it `map` overshadows the actual `map` (the one without `throws`) ... this has got to be a bug